src/invokeArgs.js

```diff
@@ -0,0 +1,36 @@
+import { curryN } from 'ramda';
+
+/**
+ * Invokes the method at path of object with given arguments.
+ * If method doesn't exists the function returns undefined.
```

Comment:
Let's drop this seconds sentence. It's an implementation detail, implied and expected.
```suggestion
```